Update Style of the Search Box on the Landing Page

Catherinesjkim opened the pull request:
Locations:
- SearchForm.js

Details:
1. Updated the search/input box to make it look more presentable for the stakeholder meeting tonight.

Catherinesjkim committed:
Add Updated Files
Locations:
- SearchForm.js
Details:
1. Updated the search/input box to make is look more presentable.

JeffreyAsuncion reviewed (dismissed):
Great work. Keep it up.

kimly143 reviewed (dismissed):
Thanks for fixing the styles

Catherinesjkim committed:
Add Updated Files
Locations:
package.json

Details:
1) Deleted babel-loader 8.1.0 from package.json file to get rid of the test error message on the PR.

JeffreyAsuncion reviewed (approved):
Great work. Keep it up.

vetorobert3 reviewed (dismissed):
Catherine, I see the fix for that annoying limit warning. Great job! also noticed the webpack dependency and various changes to styling.

vetorobert3 reviewed (commented):
I noticed the failed test. Says not to worry about the babel dependency, because it will update itself. I'll take it's word for it.

Catherinesjkim commented:
> I noticed the failed test. Says not to worry about the babel dependency, because it will update itself. I'll take it's word for it.

Thanks! I think you need to approve it for me to merge it...

JeffreyAsuncion reviewed (dismissed):
Awesome. Final stretch.

tinomen commented:
This PR is humungous. Did anyone actually review the changes?

tinomen commented on amplify.yml:
```diff
@@ -1,19 +0,0 @@
-version: 1
```
Why on earth is this file being removed?

tinomen replied:
This file needs to be restored so it doesn't break the aws amplify deploy

Catherinesjkim replied:
- Restored amplify.yml file so it doesn't break the aws amplify deployment
- Grabbed the package.json from the main branch and re-installed the new modules.

tinomen reviewed (changes requested):
You have a problem with the package.json and package-lock.json files. I suggest grabbing the files from the main branch and re-install your new modules

Catherinesjkim committed:
Add Updated Files
Locations:
- amplify.yml
- package-lock.json
- package.json

Details:
- Restored amplify.yml file so it doesn't break the aws amplify deployment
- Grabbed the package.json from the main branch and re-installed the new modules.

Catherinesjkim committed:
Add Updated Files
### Locations
- craco.config.js
- package.json
### Details
1) Restored craco.config.js
2) Restored package.json the way I had it because the one from main branch throws a lot of errors.Updated from craco to node under scripts.

JeffreyAsuncion reviewed (dismissed):
Keep up the Hard work.

Catherinesjkim committed:
Add Updated Files
### Locations
- package-lock.json
- package.json
- SearchForm.js

### Details
- Fixed problems with babel loader, eslint & webpack in react-create-app
- Restored package.json from main branch so that the design goes back to before merging my last PR and doesn't look wonky.

kimly143 commented on src/components/common/SearchForm.js:
```diff
@@ -14,10 +14,7 @@ const ColStyle = {
-  maxWidth: '80rem',
-  padding: '1rem',
+  width: '600p',
 };
```
600p ? 
p is not a unit.

Catherinesjkim replied:
Just updated the typo.

kimly143 reviewed (commented):
I think you make a typo on style.

JeffreyAsuncion reviewed (dismissed):
Way to go. Keep up the Hard work.

Catherinesjkim committed:
Add Updated File
### Locations
- SearchForm.js

### Detail
1) Updated the typo to 600px.

Catherinesjkim committed:
Add Updated Styles
### Locations
- SearchForm.js
- RenderHomePage.js

### Details
1) Updated SearchForm style so that top and length aligns well with the input style.
2) Updated RenderHomePage so that the top of the banner aligns well with the top photos.

tinomen reviewed (approved):
It's tough when adding new npm modules and the package-lock file has a crazy amount of changes. You can give a warning on the PR description when that happens.

kimly143 reviewed (approved):
Style changed and updated package json
